lib/components/Button/Button.js

```diff
@@ -47,7 +50,7 @@ const Button = forwardRef(({ variant, size, fullWidth, danger, children, ...butt
 
   return (
     // eslint-disable-next-line react/button-has-type, react/jsx-props-no-spreading
-    <button ref={ref} className={buttonClasses} {...buttonProps}>
+    <button ref={ref} className={buttonClasses} {...buttonProps} data-testid={testIds._}>
```

Comment:
I would like to see more real-world examples

Reply:
Sure. I meant to make this a draft.